dsn.go

```diff
@@ -41,10 +41,6 @@ const (
 	// a valid public key contained within its URL
 	ErrMissingPublicKey = ErrType("sentry: missing public key")
 
-	// ErrMissingPrivateKey is returned when a DSN does not have
-	// a valid private key contained within its URL
-	ErrMissingPrivateKey = ErrType("sentry: missing private key")
-
```

Comment:
Do you mind leaving this in for now and marking it as deprecated in the comment? That way we can keep the same library API and avoid breaking anybody's code when we release this. Just a comment stating that the error is never thrown since Sentry 9 has deprecated the secret key requirement should be enough.

Reply:
Good point @SPARTAN563, this change is now made. Thanks for the open-source library ;)

Reply:
Great stuff, thank you :+1:. It's an absolute pleasure, as always